test/command/test_fluentd.rb

```diff
@@ -877,7 +892,7 @@ def multi_workers_ready?
     end
 
     test 'invalid values are set to RUBYOPT' do
-      omit "hard to run correctly on Windows. Need to debug." if Fluent.windows?
+      omit "hard to run correctly because RUBYOPT=-r/path/to/bundler/setup is required on Windows while this test set invalid RUBYOPT" if Fluent.windows?
```

Comment:
Doesn't it work when RUBYOPT is  `-r/path/to/bundler/setup a`?

Reply:
It doesn't work since the invalid option `a` is also passed:

```
can't find gem bundler (>= 0.a) with executable bundle (Gem::GemNotFoundException)
	from C:/tools/ruby25/lib/ruby/2.5.0/rubygems.rb:308:in `activate_bin_path'
	from C:/tools/ruby25/bin/bundle:23:in `<main>'
```

Reply:
Understood.

Reply:
Sorry, the error log was wrong. The correct one is:

```
C:/Users/aho/Projects/fluentd/test/command/test_fluentd.rb:162:in `assert_log_matches'
Failure: test: invalid values are set to RUBYOPT(TestFluentdCommand::configured to run 2 workers):
  log doesn't match.
  <"C:\\tools\\ruby25\\bin\\ruby.exe: invalid switch in RUBYOPT: -a (RuntimeError)\r\n">
  was expected to include:
  <["Invalid option is passed to RUBYOPT"]>
```